common/utils.js

```diff
@@ -625,6 +636,13 @@
             return /*@cc_on!@*/false || !!document.documentMode;
         }
 
+        // takes URL path and returns app name
+        function appNamefromUrlPathname(path){
```

Comment:
Add a comment stating what the expected format for this function is. If I supply a path that includes filters, this won't work.